src/Tizen.NUI/src/public/Common/Vector4.cs

```diff
@@ -638,7 +638,7 @@ public float this[uint index]
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static implicit operator Vector4(float value)
         {
-            return new Vector4(value, value, value, value);
+            return ObjectPool.NewVector4(value, value, value, value);
```

Comment:
I think we cannot assume that `implicit operator Vector4(float value)` API called only at main thread.

ObjectPool is not thread safe class. So we MUST not change like this.

Reply:
Done.